.github/wiki/FirstPullRequest.md

```diff
@@ -0,0 +1,120 @@
+# Your First Pull Request, a Step-by-Step Guide
+
+These instructions provide the step-by-step process to set up your machine to make contributions to the core React Native repository, and create your first pull request.
+
+## Prologue: Getting Ready
+
+You will need a few tools and dependencies in order to build and develop for React Native. These are covered as part of the [Getting Started](https://facebook.github.io/react-native/docs/getting-started) guide under the "Building Projects with Native Code" section.
+
+In order to accept your pull request, we need you to submit a CLA. You only need to do this once to work on any of Facebook's open source projects.
+
+Complete your CLA here: <https://code.facebook.com/cla>
+
+## Chapter I: Welcome to Open Source
+
+### 1. Install `git`
+
+The React Native source code is hosted on GitHub. You can interact with the git version control through the `git` command line program. We recommend you follow [GitHub's instructions][git] to set up git on your machine.
+
+[git]: https://help.github.com/articles/set-up-git/
+
+### 2. Get the source code
+
+While you can browse the source code for React Native on [GitHub][facebook/react-native], we recommend you set up a fork on your local machine.
+
+1. Go to <https://github.com/facebook/react-native>.
+2. Click on Fork on the upper right.
+3. When asked, select your username as the host for this fork.
+
+You will now have a fork of React Native on GitHub at <https://github.com/your_username/react-native>. Next, you will grab a copy of the source code for your local machine. Open a shell and type the following commands:
+
+```bash
+git clone https://github.com/facebook/react-native.git
+git remote add fork https://github.com/your_username/react-native.git
+```
+
+> If the above seems new to you, do not be scared. You can access a shell through the Terminal application on macOS and linux, or PowerShell on Windows.
+
+A new `react-native/` directory will be created with the contents of the core React Native repository. This directory is actually a clone of the React Native git repository. It is set up with two remotes: `origin` for the upstream @facebook/react-native repository, and `fork` for the fork of React Native on your own GitHub account.
+
+### 3. Create a branch
+
+We recommend creating a new branch in your fork to keep track of your changes:
+
+```bash
+git checkout --branch my_feature_branch --track origin/master
+```
+
+## Chapter II: Implementing your Changes
+
+### 1. Make changes to the code
+
+You can now make any changes deemed necessary using your code editor of choice. Many popular editors integrate with source control in some way. You can also use `git status` and `git diff` on the command line to keep track of what has changed.
```

Comment:
What do you think about having 4 sections here:

1. Make changes to the code
2. Test your changes
3. Lint your code
4. **View your changes (NEW)**

Then, section 1 can describe how to make changes (maybe with some links to resources) and section 4 can outline viewing the diff